BREAKING.md

```diff
@@ -19,6 +19,12 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 
 ## 2.0.0-internal.5.0.0 Upcoming changes
 
+-   [IFluidResolvedUrl Deprecated](#IFluidResolvedUrl-Deprecated)
+
+### IFluidResolvedUrl Deprecated
+
+IFluidResolvedUrl is now deprecated, all usages should move to IResolvedUrl instead. This aligns with the preceding change, [IResolvedUrl equivalent to IFluidResolvedUrl](#IResolvedUrl-equivalent-to-IFluidResolvedUrl)
```

Comment:
In addition to the upcoming deprecation, you're changing various public API signatures in this PR to use/return IResolvedUrl - should update the "IResolvedUrl equivalent to IFluidResolvedUrl" breaking change to note this.

Reply:
this change no longer updates public signatures. It was safe to do, but better to do in the later removal PR.